### ISSUE TITLE
Screen goes black sometimes

### ISSUE THREAD

**Issue by CaptainOrangeJuice** [Renardjojo/PetForDesktop#91]
## Description
Occasionally, when the program is running, the screen suddenly goes black, with only the fox on top.

## Version
 - OS: Windows 11
 - Version: v1.4.1

## How to Reproduce
Steps to reproduce the behaviour:
There are none, it just happens sometimes. It also happens more consistently when I open the program.

## Expected behaviour
I expected it not to go black

Also, if I exit the program through right-clicking on the fox (but this, of course closes the program) or open a system dialogue (such as through opening Cheat Engine), then the black screen goes away. If I open the dialogue, then the fox stays.

**Comment by Renardjojo** [comment]
Indeed, I am working on this issue but it seems to be a window driver issue. I am trying to find a workaround but it can take time (swap from glfw to sdl for example)
